front/src/applications/operationalStudies/components/Study/AddOrEditStudyModal.tsx

```diff
@@ -165,42 +209,48 @@ export default function AddOrEditStudyModal({ editionMode, study, getStudy }: Pr
             <div className="row">
               <div className="col-xl-6">
                 <div className="study-edition-modal-type mb-2">
-                  <SelectImprovedSNCF
-                    title={
-                      <div className="d-flex align-items-center">
-                        <span className="mr-2">
-                          <RiQuestionLine />
-                        </span>
-                        {t('studyType')}
-                      </div>
-                    }
-                    selectedValue={{
-                      key: currentStudy?.study_type,
-                      value: t(`studyCategories.${currentStudy?.study_type || 'nothingSelected'}`),
-                    }}
-                    options={studyCategories}
-                    onChange={(e) => setCurrentStudy({ ...currentStudy, study_type: e.key })}
-                  />
+                  {studyCategories && (
```

Comment:
We need to check that studyCategories is not empty (because by default, it is [], so the current condition will always be true
```suggestion
                  {!isEmpty(studyCategories) && (
```